README.md

```diff
@@ -42,35 +42,9 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
+### Initial setup Backend
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+@Igor
 
```

Comment:
### For backend
install node.js (https://nodejs.org/en/)
### You can download the MongoDB Community Server from the MongoDB download page. 
The download is a zip file. Unzip the contents, change the folder name to “mongodb”, and 
move it to your users home directory. From there, create a “mongodb-data” directory in 
your user directory to store the database data. 
You can start the server using the following command. Make sure to swap out 
“/Users/Igor/” with the correct path to your users home directory. 

/Users/Igor/mongodb/bin/mongod --dbpath=/Users/Igor/mongodb-data 

Should run in one terminal all the time!!!!
### Install localy
npm i mongoose@5.3.16
npm i validator@10.9.0
npm i bcryptjs@2.4.3
npm i jsonwebtoken@8.4.0 
### Run in another terminal
npm run dev